LocalSettings.php

```diff
@@ -2975,6 +2975,71 @@
 			'amc' => true,
 		],
 	],
+	'wgMFQueryPropModules' => [
+		'default' => [ 
+			'pageprops', 
+		],
+		'gratisdatawiki' => [ 
+			'entityterms',
+		],
+	],
+	'wgMFSearchAPIParams' => [
+		'default' => [
+			'ppprop' => 'displaytitle',
+		],
+		'gratisdatawiki' => [
+			'wbetterms' => 'label',
+		],
+	],
+	'wgMFSearchGenerator' => [
+		'default' => [
+			'name' => 'prefixsearch',
+			'prefix' => 'ps',
+		],
+		'gratisdatawiki' => [
+			'name' => 'wbsearch',
+			'prefix' => 'wbs',
+		],
+	],
+	'wgMFVaryOnUA' => [
```

Comment:
Can you please provide an explanation for your reasoning for this one?

Reply:
> CAUTION: Setting this to true in combination with a (frontend)caching layer (such as Varnish) can have a huge impact on how your caching works, as it now caches every single page multiple times for any possible/different User Agent string!

And we do use Varnish...

Reply:
> > WARNING: You should set this to true only, if you know what you're doing!
> 
> Can you please provide an explanation for your reasoning for this one?
And this is the reason why I'm enabling it on Gratispaideia, the statement you quoted below.

Reply:
It says that on README docs. As for the other thing with Varnish, we really shouldn't do that. I think that wouldn't be great either.

Reply:
> It says that on README docs. As for the other thing with Varnish, we really shouldn't do that. I think that wouldn't be great either.

Setting this true isn't a problem at all as far as I have read, plus, caching every single page multiple times isn't alarming at all.

Reply:
There is multiple warnings about this variable on docs. So i'm not comfortable deploying it if you can't provide a good explanation for needing it.

Reply:
> There is multiple warnings about this variable on docs. So i'm not comfortable deploying it if you can't provide a good explanation for needing it.

The fact that it will improve the seo of the wiki is the reason it should be deployed, as a matter of fact, that's one aim of an Encyclopedia (doing well in seo)

Reply:
This should be removed from this PR, if you wish it to be merged in a timely manner. I'm not comfortable with it with all the warnings, and as such won't be comfortable with deploying this at all, at this time, with this setting.